snippets/getMeridiemOfInteger.md

```diff
@@ -0,0 +1,15 @@
+### toMeridiem
```

Comment:
As stated in #538 I suggest this and `to12hour` are merged into one snippet that converts to 12h format with period appended optionally. Something like this:
```js
const to12h = (hours, period = true) =>
  (hours% 12 === 0 ? 12 : hours% 12) + (period ? hours> 11 ? 'pm' : 'am' : 0); 
```

Reply:
Sure if you'd prefer that. I wrote most of these functionally so they'd call one another when that action was needed, but I removed their dependencies when submitting them.